composer.json

```diff
@@ -5,7 +5,8 @@
   "require": {
     "drewm/mailchimp-api": "^2.5",
     "constantcontact/constantcontact": "2.1.*",
-    "campaignmonitor/createsend-php": "^6.1"
+    "campaignmonitor/createsend-php": "^6.1",
+    "xwp/wp-dev-lib": "^1.6"
```

Comment:
I can't see that this library is used somewhere 🤔

Reply:
_Oh_ this one I can remove i think. The Automattic / PHP automagic commit lint thing was referring to some lint rules defined in this library, which just didn't exist on my end. So I pulled it down to get passed the code sniffer. I can delete this if ya'll don't think you need it.

Reply:
Weird, it is used in the pre-commit hook in our [other project/s](https://github.com/Automattic/newspack-plugin/blob/98ee47bdee1222b3985bfde9f5f9f20021d088a8/composer.json#L33), but not in this one. I think it can be deleted, unless I'm missing something here.